### PR TITLE
doc: translate_resource&tools

### PR DESCRIPTION
**Pull request by kkdbg666**

### CODE REVIEW

**Review by lslzl3000 — APPROVED** [review]

File: docs/guide/gui/Readme.md [hunk 1]
```diff
@@ -1,30 +1,31 @@
...
-- ViewSpace模式：在这种模式下，UI在屏幕空间中，不随3D相机的更改而变动；
-- WorldSpace模式：在这种模式下，UI可看做三维空间的一块画布，拥有3D属性（旋转、缩放、平移），能够参与深度检测等，实现与其他对象遮挡和被遮挡关系。
+Currently, `GUISpace` supports two modes: `ViewSpace` and `WorldSpace`：
 
-引擎内置有Canvas `guiCanvas` 节点作为GUI的根节点，用户的每一个面板`UIPanel`都将放入到Canvas节点中被搜集用于UI绘制；
-每个UIPanel可以视为面板根节点，在Panel下可以组织各种UI组件，用于展示UI细节。
+- ViewSpace mode: In this mode, the UI is in screen space and does not change with changes to the 3D camera;
+- WorldSpace mode: In this mode, the UI can be seen as a canvas in 3D space, with 3D attributes (rotation, scaling, translation), and can participate in depth detection, achieving occlusion and occlusion relationships with other objects.
+
+The engine has a built-in Canvas `guiCanvas` node as the root node of the GUI, and each user's panel`UIPanel`will be placed in the Canvas node and collected for UI rendering;
```
Comment (Noah133552) [inline docs/guide/gui/Readme.md:16]: each user or each panel

File: docs/guide/gui/button.md [hunk 2]
```diff
@@ -1,30 +1,30 @@
 # UIButton
 
-[UIButton](/api/classes/UIButton.md) 继承自 [UIInteractive](/api/classes/UIInteractive)（可交互组件）, 实现基本的按钮功能。 按钮组件内部封装有 `UIImage` 用于显示背景图，集成自可交互组件 `UIInteractive`，使用内置的 `UITransform` 描述的方形区域作为感应热区。
+[UIButton](/api/classes/UIButton.md) inherits from [UIInteractive](/api/classes/UIInteractive)(an interactive component), implementing basic button functions. The button component is internally encapsulated with `UIImage` for displaying background images, integrated with the interactive component `UIInteractive`, and uses the square area described by the built-in `UITransform` as the sensing heat zone.
```
Comment (Noah133552) [inline docs/guide/gui/button.md:3]: sensing heat zone can use the hotspot to replace, I double-checked the use of hotspots in UI design guidelines, such as Ant design

File: docs/guide/gui/image.md [hunk 3]
```diff
@@ -1,56 +1,56 @@
...
+[UIImage](/api/classes/UIImage) component provides image display functionality. Like the font component, the image component relies on the `Atlas` atlas. You need to first load the sprite atlas into the engine, and then assign the sprite to the image component for rendering.
 
-> 默认 `UIImage` 组件未设置精灵，将以白色方块形式呈现
+> The default `UIImage` component has no sprite set and will be presented in white squares
 
 ```ts
 import { Engine3D } from '@orillusion/core';
 
-// 创建用于显示UI的面板
+// Create a panel for displaying the UI
```
Comment (Noah133552) [inline docs/guide/gui/image.md:10]: create---adding---create---loading---...
perhaps to  use the same format, verb or verb-ing

File: docs/guide/advanced/post_ssr.md [hunk 4]
```diff
@@ -1,36 +1,37 @@
 ---
 aside: false
 ---
-# 屏幕空间反射 - SSR
-一种基于屏幕空间的反射效果实现，模拟光滑物体表面可以实时反射其周边物体影像的视觉效果。该反射效果优点为实时渲染，某一个物体发生移动，反射画面中物体也会发生移动；能精确的从每个像素反射。缺点为不能够反射出物体的背面，且屏幕范围外的物体也不能够被反射到其他的物体上。
+# SSR
+SSR is a screen-space reflection effect that simulates the visual effect of smooth surfaces reflecting the surrounding objects' images in real-time. The reflection effect has the advantages of real-time rendering, where when an object moves, the reflected objects in the image also move, and it can reflect precisely from each pixel. The disadvantage is that it cannot reflect the backside of an object, and objects outside the screen cannot reflect onto other objects.
```
Comment (Noah133552) [inline docs/guide/advanced/post_ssr.md:5]: where when
reconsider the second sentence

File: docs/guide/advanced/shader/shader_compute.md [hunk 5]
```diff
@@ -154,37 +154,37 @@ this.mRTFrame = new RTFrame([
...
 ]);
 
-// 将该纹理关联到ComputeShader
+// Associate the texture with the ComputeShader
 this.mGaussianBlurShader.setStorageTexture(`resultTex`, this.mBlurResultTexture);
 ```
 
-到这里，`ComputeShader`的初始化，相关 `GPU Buffer` 和 `Texture` 的创建与关联都已完成，接下来是执行 `ComputeShader`，在执行之前，我们还需要根据需求设置好派发调度时工作组数量，也就是参数 `workerSizeX`、
-`workerSizeY`、`workerSizeZ`：
+At this point, the initialization of the`ComputeShader`, creation and association of relevant `GPU Buffer` and `Texture` have been completed. Next, we need to execute the `ComputeShader`. Before executing, we need to set the number of workgroups according to the requirements, which are the parameters `workerSizeX`,
```
Comment (Noah133552) [inline docs/guide/advanced/shader/shader_compute.md:161]: and the creation and the association of ...
or
the initialization of xxx has been finished, and the creation and xxxx have been completed

File: docs/guide/tools/math.md [hunk 6]
```diff
@@ -181,220 +181,220 @@ let result4 = r1.innerArea(r2);
...
+// Get color value string
 let result1 = c1.getHex();
 
 ```
 
 
-## 四元数
-四元数是简单的超复数，而在图形引擎中，四元数主要用于三维旋转(四元数于三维旋转的关系)，能够表示旋转的不止四元数，还有欧拉角、轴角、矩阵等形式，之所以选择四元数，主要有以下几个优势：
+## Quaternion
+Quaternions are simple hypercomplex number numbers. In the graphics engine, quaternions are mainly used for 3D rotation (the relationship between quaternions and 3D rotation). They can represent not only quaternions but also Euler angles, axis angles, matrices and other forms of rotation. Quaternions have the following advantages:
```
Comment (Noah133552) [inline docs/guide/tools/math.md:221]: remove number

**Review by Noah133552 — APPROVED** [review]

**Pull request merged**